xero-app-store.yaml

```diff
@@ -91,6 +91,78 @@ paths:
           description: "When a failure occurs in the endpoint"
       security:
         - OAuth2: [marketplace.billing]
+  "/subscriptions/{subscriptionId}/items/{subscriptionItemId}/usage-records":
+    post:
+      tags:
+        - AppStore
+      operationId: postUsageRecords
+      summary: "Send metered usage belonging to this subscription and subscription item"
+      parameters:
+        - $ref: "#/components/parameters/subscriptionId"
+        - $ref: "#/components/parameters/subscriptionItemId"
+      responses:
+        "200":
+          description: Success - return response of the record submitted
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UsageRecord"
+        "404":
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ProblemDetails"
+          description: "When a failure occurs in the endpoint"
```

Comment:
Can we have a more meaningful description for 404 please as "When a failure occurs in the endpoint" is too generic for 404 response.

Reply:
Copied from above. I will leave this as a change for later to hammer out errors properly
https://github.com/XeroAPI/Xero-OpenAPI/blob/c90adb64639aaf1974a6cf6f58d192feee7ff599/xero-app-store.yaml#L86-L91

Reply:
I'm fine with that. It would be good to have this corrected as we are touching it already. Otherwise we might forget.

Reply:
We can revisit this at a later time. Looking forward to the Jira ticket that would revisit this.